### PR TITLE
MC-14863 Update Varnish version number

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

Magento 2.3.3 is updating the Varnish version we support. This PR updates that number.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=5047) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=5047) before we can accept your contribution.<br/>**9** out of **10** committers have signed the CLA.<br/><br/>:white_check_mark: AlexMaxHorkun<br/>:white_check_mark: belbiy<br/>:white_check_mark: VitaliyBoyko<br/>:white_check_mark: vovsky<br/>:white_check_mark: jeff-matthews<br/>:white_check_mark: eduard13<br/>:white_check_mark: dobooth<br/>:white_check_mark: keharper<br/>:white_check_mark: kirtinariya1<br/>:x: Kirti Nariya<hr/>**Kirti Nariya** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.3/graphql/caching.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Magento explicitly disallows caching the following queries.
 
 We recommend setting up Varnish as a reverse proxy to serve the full page cache in a production environment. See [Configure and use Varnish]({{page.baseurl}}/config-guide/varnish/config-varnish.html) for more information.
 
-Magento 2.3.2 updates the `vcl_hash` subroutine in the template `varnish.vcl` file and creates the `process_graphql_headers` subroutine for both Varnish 4.x and 5.x. To enable GraphQL caching, either generate a new template file, or edit the `default.vcl` file on your system to match the current default template.
+Magento 2.3.2 updates the `vcl_hash` subroutine in the template `varnish.vcl` file and creates the `process_graphql_headers` subroutine for both Varnish 4.x and 5.x. (Magento 2.3.3 supports Varnish 6.2). To enable GraphQL caching, either generate a new template file, or edit the `default.vcl` file on your system to match the current default template.
```
Comment (jeff-matthews) [inline guides/v2.3/graphql/caching.md]:
```suggestion
Magento 2.3.2 updates the `vcl_hash` subroutine in the template `varnish.vcl` file and creates the `process_graphql_headers` subroutine for both Varnish 4.x and 5.x. Magento 2.3.3 supports Varnish 6.2. To enable GraphQL caching, either generate a new template file, or edit the `default.vcl` file on your system to match the current default template.
```

File: guides/v2.3/graphql/caching.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Magento explicitly disallows caching the following queries.
 
 We recommend setting up Varnish as a reverse proxy to serve the full page cache in a production environment. See [Configure and use Varnish]({{page.baseurl}}/config-guide/varnish/config-varnish.html) for more information.
 
-Magento 2.3.2 updates the `vcl_hash` subroutine in the template `varnish.vcl` file and creates the `process_graphql_headers` subroutine for both Varnish 4.x and 5.x. To enable GraphQL caching, either generate a new template file, or edit the `default.vcl` file on your system to match the current default template.
+Magento 2.3.2 updates the `vcl_hash` subroutine in the template `varnish.vcl` file and creates the `process_graphql_headers` subroutine for both Varnish 4.x and 5.x. Magento 2.3.3 supports Varnish 6.2. To enable GraphQL caching, either generate a new template file, or edit the `default.vcl` file on your system to match the current default template.
```
Comment (keharper) [inline guides/v2.3/graphql/caching.md]: T
```suggestion
As of Magento 2.3.2, Magento supports GraphQL caching with Varnish. If you are upgrading from a previous version, you can enable GraphQL caching by generating a new template file, or by editing the `default.vcl` file on your system to match the current default template for your version of Varnish.
```

Comment (keharper) [inline guides/v2.3/graphql/caching.md]:
```suggestion
As of Magento 2.3.2, Magento supports GraphQL caching with Varnish. If you are upgrading from a previous version, you can enable GraphQL caching by generating a new template file, or by editing the `default.vcl` file on your system to match the current default template for your version of Varnish.
```

**Comment by dobooth** [comment]
running tests

**Pull request merged**